packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2809,6 +2835,12 @@ class _WidgetInspectorState extends State<WidgetInspector> with WidgetsBindingOb
   /// as selecting the edge of the bounding box.
   static const double _edgeHitMargin = 2.0;
 
+  ValueNotifier<bool> get _defaultInspectorTapBehaviorEnabledNotifier =>
```

Comment:
same comment here wrt making the naming more clear about what the behavior is. The name + the boolean value should be make the expectation clear

Reply:
Done, now `_selectionOnTapEnabledNotifier`